docs/components/base/_index.md

```diff
@@ -81,6 +80,7 @@ The base component supports the following methods:
 | [SetPower](#setpower) | Set the relative power (out of max power) for linear and angular propulsion of the base. |
 | [SetVelocity](#setvelocity) | Set the linear and angular velocity of the base. |
 | [Stop](#stop) | Stop the base. |
+| [GetProperties](#getproperties) | Get the width and turning radius of the base. |
```

Comment:
do you want to add the units here as well?

Reply:
Yeah I was going back and forth I can do that

Reply:
done!